getDeps.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
```

Comment:
Please add:
- Comments explaining this only needs to be run by **developers** of this project when moving up to a new version of SDO.
- A required argument that is the base version, for example `1.10`. (The reason for not making this `1.10.0` is that when Intel does maintenance they only increase the version of the tar files they actually changed. I.e. some could be `1.10.1` and some still `1.10.0`)